src/components/cardCarousel/CardCarousel.styles.js

```diff
@@ -3,13 +3,15 @@ import styled from "styled-components";
 export const Cards = styled.section`
   grid-area: cards;
   justify-self: center;
-  height: 70vh;
-  width: 90%;
+  height: 85vh;
+  width: 920px;
```

Comment:
Cuidado ao usar pixel. Nos componentes que estou trabalhando estou tentando ao máximo evitar e usar sempre % ou REM. Para tanto, estou usando o seguinte conversor: https://nekocalc.com/px-to-rem-converter